source/core/slang-io.cpp

```diff
@@ -316,6 +350,8 @@ namespace Slang
         ::free(absPath);
         return SLANG_OK;
 #else
+#   if 1
```

Comment:
Is `#if  1` supposed to be here? From the diff it looks like all the added code below will never actually run.